content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,301 @@
+
+Client application developers will need to work with healthcare organizations to be granted access to the Group $export operation and to be provided with Groups to be exported. Organizations can manage Groups within [Ignite Management Tooling](https://cernercentral.com/ignite-management).
+
+_Notes_
+
+* Authentication is required, open endpoints do not support bulk export workflows
+* Cerner supports v1.0.1 of the [FHIR Bulk Data Access (FLATFHIR) Implementation Guide](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/)
+* Bulk data runs against an organization's database and utilizes the same services running other applications, so it is important to consider performance. Bulk data exports large amounts of data for large groups of patients, which takes more time to complete the larger the data set or patient population.
+
+### When to use bulk data
+
+The Bulk Data Export API enables connecting applications to mine large amounts of data; however, its inherent flexibility also presents challenges. When implementing an interface between a Cerner Millennium domain and a bulk data app, it is very important that the queries inherent in the bulk data request capture the data necessary for the app – but little or no more. Furthermore, export files should be deleted as soon as they are no longer needed.
+
+There are several factors that affect the amount of time it takes to complete a Bulk API export:
+
+* The number of patients to whom the request applies (that is, the Group)
+* The number of resource types requested
+* The time range of activity being requested
+* Inherent latency due to network issues and effects from concurrent requests to the system
+
+#### Example use cases
+
+* Creating an internal clinical data warehouse for study purposes
+* Consolidation of claims data for comprehensive view
+* Machine learning applications obtaining training data from EHR
+* Integration of population health system with an EHR system
+* Transferring records between EHRs
+* Data transfer to payer database for care quality assessment
+* Submission to reportable disease (or other) registries
+
+### Best Practices
+
+* Limit group sizes to around a thousand patients or fewer.
+* Use the `_type` parameter whenever possible to improve response time and minimize storage requirements. `_typeFilter` may also be used to further refine the results to be returned.
+* Use the `_since` parameter to limit the export to only the relevant date range.
+* For groups of under one hundred patients, check the request status every ten minutes, or use exponential backoff. For groups of over one hundred patients, check the status every thirty minutes, or use exponential backoff. More information on exponential backoff can be found in the [FHIR Bulk Data Access IG](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request).
```

Comment:
need to update this line to be more generic. those recommendations listed are from EPIC and not how we will handle it.  Change to "Clients should follow an Exponential Backoff approach when polling for status as outlined in the FHIR Bulk Data Access IG."

Reply:
Updated to Fran's recommendation [here](https://github.com/cerner/fhir.cerner.com/pull/899/commits/ee37ee8e4678c8be7160f7ce7d603ef63930b8ac#diff-6963b85421ccad81bf2c26b848074843817dfa021595650e04f7ff89e1e85e31R58).